text/0069-api-v2.md

```diff
@@ -105,29 +108,20 @@ To enable API V2, which also enables the keyspace API:
 To manage keyspace:
 
 ```bash
->> config keyspaces show
->> config keyspaces create <keyspace name>
->> config keyspaces delete <keyspace name>
-
-# example: config keyspaces set-property foo default-config.raw-client.ttl-secs 100000
->> config keyspaces set-property <keyspace name> <property-path> <property-value>
+>> config keyspaces show [--deleted]
+[
+  {
+     name: string,
+     id: int64,
+     created_at: timestamp
+  }
+]
 
->> config keyspaces delete-property <keyspace name> <property-path>
->> config keyspaces flashback <new keyspace name>
+>> config keyspaces create <keyspace name> [--id <Keyspace ID>]
+>> config keyspaces delete <keyspace name>
+>> config keyspaces flashback [<new keyspace name>]
```

Comment:
Suggest describe how `flashback` is implemented.
Besides, `flashback` will delete the entry in `/keyspaces_deleted` (won't it ?), but PD doesn't provide such an API.

Reply:
updated, PTAL

Reply:
It is possible to be conflicted with another concurrent create request with 1) same name, or 2) different name but same id.

Etcd's transaction API can help to deal with the same name condition. But it would be difficult to solve the same id case.

Reply:
I think we allow multiple keyspace names referring to a same id.

Reply:
I think it would be better not to allow this.

Users would assume that two keyspaces are different if they have different names. Especially when two keyspaces have same id silently caused by the race condition. The two keyspaces will corrupt each other.